Core/include/Acts/Visualization/IVisualization.hpp

```diff
@@ -104,13 +112,35 @@ class IVisualization {
     Vector3D bd = b.template cast<double>();
     line(ad, bd, color);
   }
+
+  /// Helper: check for extension
+  ///
+  /// @note this is a placeholder for std::filesystem::has_extension
+  /// which needs special linking until gcc9
+  /// @param path the path to be checked
+  bool hasExtension(const std::string& path) const {
+    return (path.find(".") != std::string::npos);
+  }
+
+  /// Helper to replace the extension
+  ///
+  /// @note this is a placeholder for std::filesystem::has_extension
+  /// which needs special linking until gcc9
+  /// @param path [in,out] the path to be changed
+  /// @param suffix the extension to be added
+  void replaceExtension(std::string& path, const std::string& suffix) const {
+    auto ppoint = path.find_last_of(".");
+    if (ppoint != std::string::npos) {
+      path.replace(ppoint, path.length(), suffix);
+    } else {
+      path += suffix;
+    }
+  }
```

Comment:
Is there a test for this?

Reply:
Problem is, as it is now protected, I have troubles testing it ...